README.md

```diff
@@ -11,24 +11,25 @@ Web Application where people new to Machine Learning can input a dataset and exp
 Have the following installed first:
 
 1. [NodeJS v18](https://nodejs.org/en/download/) (should come with NPM v9, you must install Yarn v1.22 afterwards using NPM)
-1. [Anaconda](https://www.anaconda.com/)
+1. [Poetry](https://python-poetry.org/)
 1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html). After installing, type `aws configure` in your terminal and type in the credentials given in [Secrets](https://docs.google.com/spreadsheets/d/1fRndo-7u0MXghiZoMp3uBepDBW9EghcJ9IL4yS0TdD8/edit?usp=sharing)
 
 ## Recommended
 
 1. [GitKraken](https://help.gitkraken.com/gitkraken-client/how-to-install/) for helping with Git commands, especially if you're new to Git
 1. [Node Version Manager](https://www.freecodecamp.org/news/node-version-manager-nvm-install-guide/) for managing NodeJS versions
+1. [pyenv](https://github.com/pyenv/pyenv) for managing python versions
 
 ## To start on localhost (in order):
 
 | Action                                        | Command                                                                                    |
 | --------------------------------------------- | ------------------------------------------------------------------------------------------ |
 | Install (one-time) / Update Frontend Packages | `yarn run installf`                                                                        |
 | Install Backend Packages (one-time)           | `yarn run installb`                                                                        |
```

Comment:
@codingwithsurya can you please add poetry shell at the beginning of the yarn run installb instruction?

Reply:
I added this into `yarn run installb` @farisdurrani